packages/nocodb/src/lib/db/sql-client/lib/mssql/MssqlClient.ts

```diff
@@ -396,9 +394,7 @@ class MssqlClient extends KnexClient {
     log.api(`${_func}:args:`, args);
 
     try {
-      result.data.value = await this.sqlClient.schema.hasTable(
-        this.getTnPath(args.tn)
-      );
+      result.data.value = await this.sqlClient.schema.hasTable(args.tn);
```

Comment:
what's the reason to remove getTnPath here?

Reply:
Thank you for pointing out, this looks like a wrong change